src/qt/coincontroldialog.h

```diff
@@ -45,9 +54,15 @@ class CoinControlDialog : public QDialog
     //QAction *lockAction;
     //QAction *unlockAction;
 
+    std::pair<QString, QString> m_consolidationAddress;
+    qint64 m_consolidationAmount = 0;
```

Comment:
```suggestion
    qint64 m_consolidationAmount = 0;
```

Looks like this field isn't used.